app/jinja_filters.py

```diff
@@ -148,7 +169,7 @@ def format_datetime(context, date_time):
     return mark_safe(context, result)
 
 
-def get_format_date_range(start_date, end_date):
+def get_format_date_range(start_date: Markup, end_date: Markup) -> flask_babel:
```

Comment:
It can't just have the whole `flask_babel` library as the return type.

Reply:
Does this one not return `gettext` so `flask_babel.gettext`?

Reply:
I checked what was returned where this function was being called and it was some markup so I made this function return markup

Reply:
In my check it returns a string so should be a string or Union of string and Markup?
<img width="758" alt="Screenshot 2022-06-21 at 14 47 04" src="https://user-images.githubusercontent.com/53475968/174816060-953a6336-a75b-46fb-ac49-bd2602a58453.png">